features/step_definitions/supplier_steps.rb

```diff
@@ -100,3 +101,27 @@
   @supplier = get_supplier_with_copyable_service(@framework)
   puts "supplier id: #{@supplier['id']}"
 end
+
+Given /^I have the latest live or standstill framework$/ do
+  response = call_api(:get, "/frameworks")
+  expect(response.code).to eq(200), _error(response, "Failed getting frameworks")
+  frameworks = JSON.parse(response.body)['frameworks']
+  @framework = frameworks.select { |f| f['status'] == 'live' || f['status'] == 'standstill' }.max_by { |f| f['applicationsCloseAtUTC'] }
```

Comment:
@katstevens Currently on Preview this will match `digital-outcomes-and-specialists-4` as it is `live`. Are we OK to move `g-cloud-12` into `standstill` before this is merged?

Reply:
The test should either work or skip accordingly - can we do a check to see if there's an available esignature framework?

Reply:
@katstevens So, I think we could do a heuristic for that like `'frameworkLiveAtUTC' >= 2020-09-28` which would match `g-cloud-12` and later frameworks.  Does that make sense?

Reply:
Yep, that should do for now 👍  We can always add a TODO, to improve if we find a better way of doing this in the future...

Reply:
OK I implemented this in 5dac6d9